frontend/uddhar-frontend/src/shared/components/DashboardNavbar.jsx

```
@@ -30,12 +30,12 @@ const DashboardNavbar = ({ children, heading }) => {
           <Moon className="w-6 h-6 text-black cursor-pointer border rounded-full" />
           <Bell className="w-6 h-6 text-black cursor-pointer" />
           <div className="relative">
-            <div onClick={()=>{setUserMenuOpen(prev => !prev)}}>
+            <div onClick={()=>{setUserMenuOpen(prev => !prev)}} className="cursor-pointer">
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)